oioioi_cypress/cypress/integration/contest/settings.spec.ts

```diff
@@ -0,0 +1,71 @@
+/// <reference types="cypress" />
+
+import "cypress-file-upload";
+import { v4 as uuidv4 } from "uuid";
+
+const TYPE_OPTIONS = {
+  parseSpecialCharSequences: false,
+  delay: 0,
+};
+
+const CONTEST_TYPES = [
+  "Simple programming contest",
+  "Polish Olympiad in Informatics - Online",
+  "Polish Olympiad in Informatics - Onsite",
+  "Polish Olympiad in Informatics - Onsite - Finals",
+  "Baltic Olympiad in Informatics",
+  "Baltic Olympiad in Informatics - online",
+  "ACM style contest",
+  "ACM style contest (open)",
+];
+
+context("Access settings for contest", () => {
+  beforeEach(() => {
+    loginAsAdmin();
+    chooseEnglishLang();
+  });
+
+  CONTEST_TYPES.forEach((type: string) => {
+    it(`should access settings of ${type}`, () => {
+      const name = uuidv4().substring(10);
+      addContest(type, name);
+      accessSettings(name);
+      deleteContest(name);
+    });
+  });
+});
+
+const loginAsAdmin = () => {
+  cy.visit("/");
+  cy.get(".username").click();
+
+  cy.fixture("credentials").then((data) => {
+    cy.get("#navbar-login-input").type(data.user.username);
+    cy.get(":nth-child(4) > .form-control").type(data.user.password);
```

Comment:
is it possible to access this element by name like:
`.get("input[name='auth-password']")`

Reply:
actually why not use `cy.login(data.user)`